tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -110,8 +110,23 @@ def test(self):
         # check Meshcop-e service
         self.check_meshcop_e_service(host, True)
 
-        # deactivate ePSKc mode
-        br1.deactivate_ephemeral_key_mode()
+        # deactivate ePSKc mode in force
+        br1.deactivate_ephemeral_key_mode(retain_active_session=False)
```

Comment:
probably in separate PR: introduce ot-commissioner host and verify the scenarios when there is DTLS secure session established